src/hooks/useAnalytics.ts

```diff
@@ -40,7 +40,7 @@ export const setAnalyticsGroup = (groupKey: string, groupName: string) => {
 
 export const initAnalytics = (opts?: Object) => {
   if (!MIXPANEL_PROJECT_TOKEN) {
-    console.warn("Mixpanel is not set up correctly. The token is missing.");
```

Comment:
This being at warning level led to a lot of stack traces in the console which made it hard to debug things that were genuinely broken.

Reply:
[NOTE]
I think the cleaner solution is to change from firing regularly, to only once per load. I wonder if we should include that in the `SPARK` object and if warning/info has already been shown, not spam it again and again.